src/stories/components/breadcrumbs/customize-breadcrumbs/customize-breadcrums.stories.tsx

```diff
@@ -0,0 +1,75 @@
+import { siteRoutes } from '@ses/config/routes';
+import { createThemeModeVariants } from '@ses/core/utils/storybook/factories';
+import CustomizeBreadCrumbs from './customize-breadcrums';
+import type { ComponentMeta } from '@storybook/react';
+
+export default {
+  title: 'Components/General/CustomizeBreadCrumbs',
+  component: CustomizeBreadCrumbs,
```

Comment:
This component looks different in several breakpoints, you should add the chromatic viewports to detect those changes

Reply:
ok, done